services/web-app/utils/schema.js

```diff
@@ -130,13 +150,23 @@ export const collapseAssetGroups = (asset, filter) => {
  * @param {import('@/typedefs').Asset} asset
  * @returns {string} License
  */
-export const getLicense = (asset) => {
+export const getAssetLicense = (asset) => {
   const defaultLicense = asset.params.host === 'github.ibm.com' ? 'IBM internal' : 'No license'
   const { license = defaultLicense } = asset.library ? asset.library.content : asset.content
 
   return license
 }
 
+/**
+ * Gets the license from a designKit
+ * @param {import('@/typedefs').DesignKit} designKit
+ * @returns {string} License
+ */
+export const getDesignKitLicense = (designKit) => {
+  const license = licenses[designKit.license]?.name
+  return license
```

Comment:
I'm surprised SonarCloud doesn't complain about this wanting:

```suggestion
  return licenses[designKit.license]?.name
```

Just commenting. Doesn't need changed IMO.

Reply:
🤷🏽‍♀️